vuepress/docs/next/tutorials/solution/entando-hub.md

```diff
@@ -29,7 +29,14 @@ The Hub is installed using the Entando Component Repository (ECR) and two Entand
 - An Entando Application on any Kubernetes provider. Follow the [tutorials](../#operations) appropriate to your environment to install the Entando platform.
 - The [ent command line tool](../../docs/reference/entando-cli.md#overview), installed and connected to your Kubernetes instance.
 
-### Installation Steps
+### Automatically Access the Hub from the App Builder
+1. Login to your App Builder
+2. Got to `Registry` → `Select Registry` kebob buttons 
+3. Choose `New Registry`
+4. In the pop-up window, enter `Entando Hub` and URL [https://hub.entando.com], then `Save` 
```

Comment:
"for the URL" like above sounds a little better... comment applies to all other instances below